### PR TITLE
Updating the Documentation across various sources for workshop readiness

### PR DESCRIPTION
**Pull request by catherine-martlin**
Updated out of date websites on readthedocs main page

### CODE REVIEW

**Comment by catherine-martlin** [comment]
When we have phase_constraint in be sure to update the README and ReadTheDocs - "The Phase Constraint Calculator provides a simple interface for calculating the JWST observation start window. The calculation currently only applies to transits, though one can subtract 0.5 from the phase values to compute the eclipse observation start window for planets on circular orbits.  Enter the minimum and maximum phase values into the APT special requirements section when planning your observations. " (The current blurb for the webpage but do double check on the production version.)

**Comment by bourque** [comment]
@catherine-martlin At yesterday's rehearsal we said that we are going to try to get the 1.0 release out by the end of the day today.  Is there anything more to do on this PR?  If so, let me know if there is anything I can do to help.

**Comment by catherine-martlin** [comment]
@bourque - This is ready for review - I should have taken care of the merge conflicts with master and most changes are to the text so if you do a test on tlexo hopefully things will run the same as they do for current Master.

**Comment by bourque** [comment]
Sounds good.  I will also test out the `sphinx` build and make sure ReadTheDocs is working.

**Comment by bourque** [comment]
This all looks good when running the web app locally, and making the sphinx docs locally.

I am having trouble getting this to build in ReadTheDocs since the RTD build expects there to be an existing `EXOCTK_DATA` directory.  I'm currently trying to find clever ways to get around that.

**Comment by catherine-martlin** [comment]
Thank you for looking into that @bourque - I haven't had to test on `tlexo` yet so didn't realize that issue!

**Comment by bourque** [comment]
Ok, I think I fixed the ReadTheDocs build by modifying `utils.py` to avoid checking for the `EXOCTK_DATA` variable when being executed on the RTD servers (this is akin to checking if the code is bring run on travis servers).

To summarize:

1. This `README` looks good to me
2. The web app looks good to me when running locally
3. The web app looks good to me when running from the test server
4. The ReadTheDocs page looks good to me (now that its actually building)
5. All links everywhere seems to be working

The only thing I am a bit concerned about was that the rebasing on this branch seemed to make it appear that a lot of files are changing with the PR.  But since that rebase happened this morning, I think it is fine?  Perhaps @julesfowler can comment on that.

I would like @julesfowler to give this a look over to make sure I am not missing anything, if that is OK.  They can push the merge button if everything looks fine.

**Comment by catherine-martlin** [comment]
I definitely agree on having Jules check the rebase - I agree with you that I think all the files it's counting as changing are actually just files created in your and Jenny's PRs. The only conflicts I hit while rebasing were in the README and it was because I added in some of the info you (Matthew) then added in yourself in your PR so it was just two copies of the same info in the README.

File: env/environment-3.5.yml [hunk 1]
```diff
@@ -15,31 +16,30 @@ dependencies:
...
     - platon==4.0
-    - pysynphot==0.9.12
-    - sphinx_astropy==1.1.1
-    - sphinx-automodapi==0.11
+    - pysynphot==0.9.14
+    - sphinx_astropy==1.2
+    - sphinx-automodapi==0.12
     - svo-filters==0.2.19
-    - git+https://github.com/spacetelescope/jwst_gtvt.git
+    - git+https://github.com/spacetelescope/jwst_gtvt.git@cd6bc76f66f478eafbcc71834d3e735c73e03ed5
```
Comment (julesfowler) [inline env/environment-3.5.yml]: Why is the environment updating here?

Comment (julesfowler) [reply env/environment-3.5.yml]: Also -- why does it not update for 3.5?

File: exoctk/atmospheric_retrievals/aws_config.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+    "ec2_id" : "i-07717dc4bee75607a",
+    "ssh_file" : "/Users/bourque/.ssh/exoctk-kramer.pem"
+}
```
Comment (julesfowler) [inline exoctk/atmospheric_retrievals/aws_config.json]: Why is this here? Even if this is weird material pulled from another branch, this looks like something probably only Matthew can use and prob should not be committed.

**Comment by julesfowler** [comment]
Changes to the readme look good. I'm going to check nothing is different than master due to the rebase trickery now.

**Comment by julesfowler** [comment]
@catherine-martlin @bourque Our readthedocs still say they are running v0.4.0. What do we need to do to have it read for our 1.0 release?

File: exoctk/atmospheric_retrievals/aws_config.json (hunk 2 above)
Comment (bourque) [reply exoctk/atmospheric_retrievals/aws_config.json]: Good catch!  I changed it back to blank values.

File: env/environment-3.5.yml (hunk 1 above)
Comment (bourque) [reply env/environment-3.5.yml]: @julesfowler It is because the 3.5 build was failing, but others were not.

Comment (julesfowler) [reply env/environment-3.5.yml]: Fair enough.

**Comment by bourque** [comment]
@julesfowler We will have to update `utils.py` to use `1.0` as the version number,  instead of `0.4`, when we are ready.

File: README.rst [hunk 3]
```diff
@@ -13,8 +13,9 @@ ExoCTK is an open-source, modular data analysis package focused primarily on atm
 * Integrations and Groups Calculator
 * Transit Light-Curve Fitter
 * Limb Darkening Calculator
-* Atmopsheric Retrievals
-* Atmospheric Forward Modeling - Currently only available through the `website <https://exoctk.stsci.edu/fortney>`_
+* Atmospheric Retrievals
+* Phase Constraint Calculator
+* Atmospheric Forward Modeling - Currently only available through the `website <https://exoctk.stsci.edu/fortney>`_.
```
Comment (julesfowler) [inline README.rst]: This is no longer true, actually. Why don't we just leave it as its own bullet point like the rest.

File: README.rst [hunk 4]
```diff
@@ -13,8 +13,9 @@ ExoCTK is an open-source, modular data analysis package focused primarily on atm
...
 
 For more information on each package visit our documentation `website <https://exoctk.readthedocs.io/en/latest/>`_.
```
Comment (julesfowler) [inline README.rst]: Let's have this say "our documentation at readthedocs" -- so it doesn't say website a bunch of times.

**Comment by julesfowler** [comment]
@catherine-martlin If you could address the small content changes by tomorrow, that would be great. Or I will do so tomorrow.
